packages/monitor-v2/src/monitor-og/SnapshotVerification.ts

```diff
@@ -104,27 +111,59 @@ const getGraphqlData = async (ipfsHash: string, url: string): Promise<GraphqlDat
     }
   `);
   try {
-    return await request<GraphqlData, { ipfsHash: string }>(url, query, { ipfsHash });
-  } catch (error) {
-    return new Error(JSON.stringify(error));
+    return await retry(
```

Comment:
nit: related to the comment below, I think there's a simplification possible here that would streamline this to avoid some of the flow complexity/branching. Thoughts?
```ts
return await retry(
  () => request<GraphqlData, { ipfsHash: string }>(url, query, { ipfsHash }),
  retryOptions
).catch((error) => {
  assert(lastTryError instanceof Error, "Unexpected Error type!");
  return error;
});

Reply:
Agree, fixed as per suggestion above. Except I omit await as the method is returning promise and would be awaited in the caller.